lib/Installation/Partitioner/EnlargeToRAMSizeForSuspendCheckbox.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces all accessing methods for 'Enlarge to RAM size'
+# checkbox'.
+# The checkbox is extracted to the separate package as several pages contain
+# it, but different shortcut is used for selecting.
+
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>, Dawei Pang <dawei.pang@suse.com>
```

Comment:
> Some time ago we decided to use team mailing list for maintainer field:
> 
> ```
> Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
> ```
> 
> We still have some files with personal names but this will be changed soon.
> 
> Could you please change it to the mailing list or keep only your name in case you want to be the only one person to support the modules in case of some issues.
> 
> Same for other files in this PR where maintainers were added.

Thanks for your comments. The maintainers of new files are changed to mailing list.